modules/dag-l0/src/main/scala/io/constellationnetwork/dag/l0/http/routes/NodeParametersRoutes.scala

```diff
@@ -80,11 +142,41 @@ final case class NodeParametersRoutes[F[_]: Async](
         .map(validStateForSnapshotReturn)
         .ifM(
           getLatestNodeParameters(nodeId.toId).flatMap {
-            case Some(params) => Ok(params)
+            case Some(params) => Ok(params._1.value)
             case _            => NotFound()
           },
           ServiceUnavailable()
         )
+    case req @ GET -> Root =>
+      nodeStorage.getNodeState
+        .map(validStateForSnapshotReturn)
+        .ifM(
+          (for {
+            sortOrder <- SortOrder.fromRequest(req)
+            sort <- Sort.fromRequest(req)
+          } yield (sortOrder, sort)) match {
```

Comment:
```scala
(SortOrder.fromRequest(req), Sort.fromRequest(req)).mapN { case (sortOrder, sort) =>
   ...
}.getOrElse(BadRequest())
```
but, there is a repeated part, so it would be better to use kleisli:
```scala
(getSortOrder, getSort).mapN {
  case (sortOrder, sort) => ...
}
  .run(req)
  .getOrElse(BadRequest())
```